.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
If you're interested in commit hooks, surely ok to add it. But don't feel forced to use black, isort, commit hooks, versioneer if you prefer not to use them. They may make more sense in bigger projects, for something smaller like a backend not sure if they add value, or they just become annoying. Up to you.

If you do use black here, better add it to the CI and be consistent.

Reply:
I think it makes sense to have a hooks that match the CI for people that choose to use pre-commit. I don't have particularly strong feelings on versioneer, etc, but it makes sense to me to default to what the core framework uses. 

I'll add black to the CI.